src/components/shared/ManagedTree.js

```diff
@@ -127,8 +127,14 @@ class ManagedTree extends Component<Props, State> {
       // closed folder and highlights this folder
       const index = highlightItems
         .reverse()
-        .findIndex(item => !expanded.has(this.props.getPath(item)));
-      this.focusItem(highlightItems[index]);
+        .findIndex(
+          item =>
+            !expanded.has(this.props.getPath(item)) && item.name !== "root"
+        );
+
+      if (highlightItems[index]) {
+        this.focusItem(highlightItems[index]);
+      }
```

Comment:
We now have lots of good tests for SourcesTree
https://github.com/devtools-html/debugger.html/blob/master/src/components/PrimaryPanes/SourcesTree.js#L184

It might be nice to add a new passing test for the non-root case and perhaps add a failing test for the root case so that we have documented this behavior